.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  run-tests:
+    continue-on-error: true
+    runs-on: [ self-hosted ]
+    strategy:
+      matrix:
+        python-version: ['3.8']
+        django-version: ["django22", "django30", "django31", "django32"]
+        test_module: [
```

Comment:
For future we need make sure newly added apps appear here as well. One idea might a test case which compare these harded apps with platform lms cms etc apps.

Reply:
1. Could we have an action that just generates this list and creates an update PR if needed? That way, if your check fails (whatever it would be), it could provide a simple action that would resolve and keep this ordered appropriately.
2. Also, somewhere we added a check to ensure that we were collecting all tests. I think this was a result of all common tests no longer working at some point? Will we still have that check, or does it need to be updated?